packages/cli/src/cli.ts

```diff
@@ -29,6 +29,10 @@ import { logVersionMiddleware } from "./yargs/logVersionMiddleware.js";
 import { YargsCheckError } from "./YargsCheckError.js";
 
 export async function cli(args: string[] = process.argv) {
+  consola.info(
+    `Palantir OSDK CLI ${process.env.PACKAGE_VERSION}`,
```

Comment:
Can we try to maintain the spacing to be consistent as well when you use `--help` or not? For example just add an extra new line
```suggestion
    `Palantir OSDK CLI ${process.env.PACKAGE_VERSION}\n`,
```

Reply:
I don't think we can make the space consistent. In all cases other than `--help` we get an extra line from `logVersionMiddleware`. If we add a new line like above we will in all cases (except for `--help`) get the following:

(non verbose)
```
ℹ Palantir OSDK CLI 0.5.0              
.
.
other logs    
```                                                                                                                                                           

(verbose)
```
ℹ Palantir OSDK CLI 0.5.0              
.
⚙ Verbose logging enabled (debug)                                                                                                                                                                     
.
other logs    
```

So I would say in its current shape its consistent for almost all cases except for the `--help`.